cove/templatetags/cove_tags.py

```diff
@@ -145,4 +145,16 @@ def html_error_msg(error):
             error.get("instance"),
         )
 
+    if e_validator == "minProperties":
+        return _("{} does not have enough properties").format(e_instance)
+
+
+    if error.get("error_id"):
```

Comment:
might be a bit more pythonic to use try and catch the keyerror?

Reply:
This also handles the case where the value exists, but is None (which does happen for this).